### DISCUSSION TITLE
Exporting in 48KHZ

### DISCUSSION THREAD

**Discussion by toady92037** [Anjok07/ultimatevocalremovergui#1248] [General]
How do I export my files in 48Khz? it seems like UVR 5 only exports files in 41Khz.

**Comment by BadaBingCherry** [comment]
Anyone on how to export 48Khz? The input files are 48 but they are saved as 44Khz. Thanks!

**Comment by laurensiussteven** [comment]
same problem here, I need export in 48khz , 24 bit tooo

**Comment by NLesha** [comment]
It is also exporting in stereo with the original files being in mono. Can that also be fixed?
